clients.json.dist

```diff
@@ -0,0 +1,7 @@
+{
+    "client-name": {
```

Comment:
JSON is easy to parse, in general I would reserve it to files that are written by programs rather than configuration as it doesn't support comments, the last element `,`, etc... I realize every format has its own defects

Reply:
Yeah, I normally use YAML. Will take a look.